documentation/source/user_section/tutorials/analysis-pipelines-with-sct/handling-issues.rst

```diff
@@ -6,4 +6,75 @@ What if things go wrong?
 
 If you spot any issues during QC, e.g., error during segmentation, you can correct the issue (manually or by adjusting SCT parameters), and then re-run the processing.
 
-``process_data.sh`` contains a number of convenience functions that will select any file with suffix ``-manual``, if it exists, then use that file (instead of automatically computing, e.g., a segmentation). That way, you can manually process outlier subjects without interrupting the automated processing for the dataset as a whole.
+Manual corrections
+==================
+
+SCT provides a downloadable script called ``manual_correction.py`` that can be used to conveniently correct any segmentations with issues (i.e. the subjects that were marked with "Fail" during :ref:`the previous QC step <batch-qc_report>`).
```

Comment:
maybe we can omit "downloadable"? Although I understand you use it because we need to run (`sct_download_data -d manual-correction -o manual-correction`), right?

```suggestion
SCT provides a script called ``manual_correction.py`` that can be used to conveniently correct any segmentations with issues (i.e. the subjects that were marked with "Fail" during :ref:`the previous QC step <batch-qc_report>`).
```

Reply:
Exactly, yes! I wanted to clarify that it's not built-in to SCT yet (e.g. `sct_manual_correction`). But definitely optional. :)

Reply:
okay, let's keep to inform users that the script needs to be downloaded